shared/profile/index.desktop.js

```diff
@@ -249,6 +249,8 @@ class ProfileRender extends PureComponent<Props, State> {
       }
     }
 
+    // TODO/songgao: is it intended that this prop is still here? The prop is
+    // not provided in the container at all.
```

Comment:
Ugh I'm realizing I accidentally deleted a previous comment: I think this is the last major use of `constants/types/folders.js`. It seems to intend to show a list of user's folders in the profile page, but isn't actually effective since the container doesn't provide the `tlfs` prop at all, thus the app currently doesn't show it either. Do we still want it shown or is it safe to remove this?